src/Chunk.h

```diff
@@ -439,7 +439,12 @@ class cChunk :
 	void SetAlwaysTicked(bool a_AlwaysTicked);
 
 	// Makes a copy of the list
-	cClientHandleList GetAllClients(void) const {return m_LoadedByClient; }
+	cClientHandleList GetAllClients(void) const
+	{
+		cClientHandleList copy;
+		std::copy(m_LoadedByClient.begin(), m_LoadedByClient.end(), std::back_inserter(copy));
+		return copy;
```

Comment:
Could someone perhaps explain the difference to me between std::copy and returning a copy of the list? Does std::copy make copies of the clienthandles?